packages/ui-bindings/EconomicEvent/CreateEconomicEvent.tsx

```diff
@@ -0,0 +1,67 @@
+import { connect } from "react-redux";
+import { gql, graphql, compose } from "react-apollo";
+import { AppState } from "@vflows/store/types";
+import { getActiveLoginToken } from "@vflows/store/selectors/auth";
+import EconomicEventFragment from "./EconomicEvent.tsx";
+
+export const mutation = gql`
+  mutation(
+    $receiverId: Int,
+    $fulfillsCommitmentId: Int,
+    $createResource: Boolean,
+    $inputOfId: Int,
+    $url: String,
+    $resourceImage: String,
```

Comment:
Should this be string or String

Reply:
That's not a Typescript String type, that's part of the GraphQL query syntax. It's specifying that resourceImage should be interpreted as a string. If this isn't what you meant, please comment again.